frame/sudo/src/lib.rs

```diff
@@ -153,14 +173,18 @@ decl_module! {
 		/// - O(1).
 		/// - The weight of this call is defined by the caller.
 		/// # </weight>
-		#[weight = (*_weight, call.get_dispatch_info().class)]
-		fn sudo_unchecked_weight(origin, call: Box<<T as Config>::Call>, _weight: Weight) -> DispatchResultWithPostInfo {
+		#[pallet::weight((*_weight, call.get_dispatch_info().class))]
+		pub(crate) fn sudo_unchecked_weight(
+			origin: OriginFor<T>,
+			call: Box<<T as Config>::Call>,
+			_weight: Weight
```

Comment:
I think code format guidelines say so
```suggestion
			_weight: Weight,
```